x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextFieldMapper.java

```diff
@@ -679,9 +681,9 @@ public static void insertValue(String path, Map<?, ?> map, Object newValue) {
         }
     }
 
-    private record SuffixMap(String suffix, Map<String, Object> map) {}
+    public record SuffixMap(String suffix, Map<String, Object> map) {}
```

Comment:
Why do we need this visibility change? 🤔

Reply:
removed, thanks